frontend/src/pages/servers/index.tsx

```diff
@@ -38,27 +62,60 @@ const getStats = (servers: Array<Essential>): Stats => ({
   servers: servers.length,
 });
 
-const Row = ({ s }: { s: Essential }) => (
-  <li className="hover-bg-black-10 lh-copy pa2 ph0-l bb b--black-10">
-    <Link href={"/servers/" + s.ip}>
-      <a className="link black flex items-center justify-between">
-        <div className="overflow-hidden">
-          <span className="db black-70 measure truncate">{s.hn}</span>
-          <span className="db black-30 f6 measure truncate">{s.gm}</span>
-        </div>
+const Row = ({ s }: { s: Essential }) => {
```

Comment:
Minor style thing, and better prop name:
```suggestion
type RowProps = { server: Essential }
const Row: FC<RowProps> = ({ server }) => {
```